code/game/objects/items/melee/misc.dm

```diff
@@ -136,6 +136,41 @@
 		user.death(FALSE)
 	REMOVE_TRAIT(src, TRAIT_NODROP, SABRE_SUICIDE_TRAIT)
 
+/obj/item/melee/sabre/mime
+	name = "Bread Blade"
+	desc = "An elegant weapon, it has an inscription on it that says:  \"La Gluten Gutter\"."
+	force = 18
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "rapier"
+	item_state = "rapier"
+	lefthand_file = null
+	righthand_file = null
+	alpha = 0
+	block_level = 1
+	block_upgrade_walk = 1
+	block_power = 60
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
+
+/obj/item/melee/sabre/mime/attack_hand(mob/user)
+	..()
+	if(user.mind.miming)
+		alpha = 255
+
+/obj/item/melee/sabre/mime/Moved(oldLoc, dir)
+	if (iscarbon(oldLoc))
+		alpha = 0
+	..()
```

Comment:
I dont really understand what this is trying to accomplish

Reply:
When the sword changes locations it goes invisible, when used in hand it becomes visible?

Reply:
I tried to make it visible to mimes only

Reply:
Ah you would likely want something like an alt appearance for that. It makes it so an image on something will only show to a specific player and is used for things like AI not being able to see cult runes and cult hallucinations.